README.md

```diff
@@ -106,9 +106,9 @@ $ cargo +nightly build -p filecoin-proofs --release --target aarch64-unknown-lin
 The main benchmarking tool is called `benchy`.  `benchy` has several subcommands, including `merkleproofs`, `prodbench`, `winning_post` and `window_post`.  You can run them with various configuration options, but some examples are below:
 
 ```
-> cargo run --release --bin benchy -- merkleproofs --size 2
-> cargo run --release --bin benchy -- winning-post --size 2
-> cargo run --release --bin benchy -- window-post --size 2
+> cargo run --release --bin benchy -- merkleproofs --size 2048
```

Comment:
There is still a typo here on `--merkleproofs`.  Verifying size change is correct currently, as at one point we intentionally multiplied by 1KiB.